src/main/webapp/js/userMapData.json

```diff
@@ -95,7 +103,8 @@
       "Bahrain Polytechnic"
     ],
     "Botswana": [
-      "Alpha"
+      "Alpha",
+      "University of Botswana"
     ],
     "Brazil": [
       "AEDB",
```

Comment:
Dear @damithc thank you for your advice. However, I cannot fully understand what you want me to do. The underlinned in red line indicates that the "Alpha" was moved 8 lines because I added the new univerisities. What else do you want me to do now?
Thank you.